backend/pkg/proto/service.go

```diff
@@ -412,15 +412,13 @@ func (s *Service) createProtoRegistry() error {
 // protoFileToDescriptorWithBinary parses a .proto file and compiles it to a descriptor using the protoc binary. Protoc must
 // be available as command or this will fail.
 // Imported dependencies (such as Protobuf timestamp) are included so that the descriptors are self-contained.
-//
-// ProtoPath is the path that contains all .proto files. This directory will be searched for imports.
-// Filename is the .proto file within the protoPath that shall be parsed.
 func (s *Service) protoFileToDescriptor(files map[string]filesystem.File) ([]*desc.FileDescriptor, error) {
 	filesStr := make(map[string]string, len(files))
 	filePaths := make([]string, 0, len(filesStr))
 	for _, file := range files {
-		filesStr[file.Path] = string(file.Payload)
-		filePaths = append(filePaths, file.Path)
+		filePath := string(file.Path)
```

Comment:
What's the reason for this variable? `file.Path` already is a string I think?

Reply:
Oh right, sorry - this is actually my first Go development at all ;)

Fixed

Reply:
Also wondering if there is not an easier way to create a sub string from it, but could not found any simpler solution